aws_xray/stackstate_checks/aws_xray/aws_xray.py

```diff
@@ -343,9 +342,31 @@ def flatten_segment(segment, kind):
     # STAC-13020: we decided to make all the spans from aws-xray as CLIENT to produce metrics
     # otherwise receiver won't produce any metrics if the kind is INTERNAL
     flat_segment["span.kind"] = kind
+    flat_segment["span.serviceType"] = "xray"
     return flat_segment
 
 
+def process_http_error(span, segment):
+    """
+    Process the span on different error code
+    :param span: The span in which error code has to be sent
+    :param segment: The segment from which error code has to be processed
+    :return: span processed with error codes
+    """
+    error = segment.get('fault') or segment.get('throttle') or segment.get('error')
```

Comment:
what is the meaning of `throttle` ?

Reply:
@TizianoPerrucci  this is the `errortype` returned by `AWS` for status code 429.